tools/torch_quant/torch_quant/graph.py

```diff
@@ -159,6 +174,28 @@ def set_qconfig(ctx: GraphModContext) -> None:
             m.qconfig = QConfig(activation=None, weight=ctx.w_ob_ctr)
 
 
+def fuse_modules(ctx: GraphModContext) -> None:
+    """ Fuse modules for quantization
+    Fuses only the following sequence of modules:
+        Linear + ReLU -> LinearRelu
+        Conv2d + ReLU -> Conv2dRelu
+    """
+    # TODO(wanchen.swc): refactor this code to support other fusion patterns
```

Comment:
add a todo for extracting generic pattern matching functions.

Reply:
done